packages/web-components/tools/babel-plugin-create-react-custom-element-type.js

```diff
@@ -30,7 +30,7 @@ function createMetadataVisitor(api) {
   const { types: t } = api;
 
   /**
-   * @param {Path} path The Babel path what a `@property()` decorator call refers to.
+   * @param {string} path The Babel path what a `@property()` decorator call refers to.
```

Comment:
`path` in Babel is of AST tree node instead of file path. What kind of ESLint error did you get...?

Reply:
@asudoh This is the lint warning

![image](https://user-images.githubusercontent.com/909118/110965770-847f6400-8322-11eb-8558-27d15a9cc522.png)

Reply:
Thanks @asudoh, I'll update the `typedef` since I don't think we should suppress valid warnings with the ignore.

Reply:
I see it's good, just in case it helps, `Path` type comes from 3rd party library (for this particular case, Babel). Given we are not on TypeScript for this particular code, we don't have a good way to refer to those 3rd party types (or we define every 3rd party types)...